transports/azure-service-bus/queue-permissions.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Endpoint Managed Entity Queue Permissions
+summary: Explains queue-scoped permissions needed by an endpoint running in Azure Service Bus
+component: ASBS
+reviewed: 2025-10-14
+---
+
+When using managed entities queue-scoped permissions for endpoint connections to Azure Service Bus the following minimum permissions are required:
+
+- `Azure Service Bus Data Receiver` to the endpoint's queue is required to process messages.
+- `Azure Service Bus Data Sender` to the endpoint's queue is required for:
```

Comment:
Is it worthwhile to be so detailed? Wouldn't it be more maintainable and future proof by documenting that a non-send-only endpoint requires receiver and sender rights to its own queue and all the subqueues like satellites or instance specific queues?

Then we could also add a more generic statement mentioning data sender is required for every queue or topic the endpoint interacts with either directly or indirectly via plugins / extensions (audit, metrics...).

Plus subscription write rights on the topics it subscribes to when autosubscription is enabled

Another angle could be to mention also that when installers are enabled more elevated rights are required because entities might get created during the startup.

Reply:
I'm OK with that reasoning